config/app-sre-build-push.sh

```diff
@@ -2,61 +2,46 @@
 
 set -ex
 
-# Usage: push.sh REGISTRY NAMESPACE NAME
-# e.g. push.sh quay.io app-sre boilerplate
+# Usage: app-sre-build-push.sh REGISTRY NAMESPACE NAME
+# e.g. app-sre-build-push push.sh quay.io app-sre boilerplate
 # Builds and pushes a new tagged image IF the most recent tag does not yet have an image.
 # Checks out the right version for the tag beforehand. 
 # Assumes $QUAY_USER and $QUAY_TOKEN are set in the env.
 
-registry=$1
-namespace=$2
-name=$3
-
-quay_image=$registry/$namespace/$name
-
-HERE=$(realpath ${0%/*})
-container_engine=${CONTAINER_ENGINE:-$(command -v podman || echo docker)}
-
-build_cumulative() {
-    local tag=$1
-    echo "Building for tag $tag"
-    $container_engine build -t ${name}:${tag} -f ${HERE}/Dockerfile.appsre ${HERE}
-}
-
-push_for_tag() {
-    local tag=$1
-    echo "Pushing for tag $tag"
-    skopeo copy --dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-        "docker-daemon:${name}:$tag" \
-        "docker://${quay_image}:$tag"
-}
-
 # Get the most recent tag starting with "image-v*"
 latest_tag=$(git describe --tags --match "image-v*" --abbrev=0)
 if [ -z ${latest_tag} ]; then
     echo "No tag matching pattern 'image-v*' found."
     exit 1
 fi
```

Comment:
/unhold

This section will allow `image-v5.0.1` to be built from the previous commit